wpilibcExamples/src/main/cpp/examples/Frisbeebot/cpp/subsystems/ShooterSubsystem.cpp

```diff
@@ -24,7 +24,8 @@ ShooterSubsystem::ShooterSubsystem()
 
 void ShooterSubsystem::UseOutput(double output, double setpoint) {
   m_shooterMotor.SetVoltage(units::volt_t{output} +
-                            m_shooterFeedforward.Calculate(kShooterTargetRPS));
+                            m_shooterFeedforward.Calculate(
+                                units::turns_per_second_t{kShooterTargetRPS}));
```

Comment:
kShooterTargetRPS should be a unit type instead of being wrapped here.

Reply:
done